File: extract_cv.py
import sys

try:
    from pypdf import PdfReader
    reader = PdfReader("Miraz_Hossain_CV (1).pdf")
    text = ""
    for page in reader.pages:
        text += page.extract_text() + "\n"
    with open("cv_text.txt", "w", encoding="utf-8") as f:
        f.write(text)
    print("Extraction complete via python.")
except Exception as e:
    print(f"Error: {e}")

File: extract_cv.js
import fs from 'fs';
import { createRequire } from 'module';
const require = createRequire(import.meta.url);
const pdf = require('pdf-parse');

let dataBuffer = fs.readFileSync('Miraz_Hossain_CV (1).pdf');

pdf(dataBuffer).then(function(data) {
    fs.writeFileSync('cv_text.txt', data.text);
    console.log("Extraction complete.");
}).catch(err => {
    console.error("Error reading PDF:", err);
});
